content/elections/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "OHBM OSSIR 2021 elections"
+subtitle: "Make a difference: join the OS SIG!"
+date: 2020-06-30
+draft: false
+sidebar: false # true or false to display the sidebar
+sidebarlogo: fresh-white-alt # From (static/images/logo/)
+include_footer: true
+---
+
+# Make a difference: join the OS SIG!
+
+We are excited to announce this year's Nominations for the OHBM open-science Special Interest Group (OS SIG) for OHBM 2021!
+
+Please self-nominate yourself [HERE](https://forms.office.com/Pages/ResponsePage.aspx?id=DQSIkWdsW0yxEjajBLZtrQAAAAAAAAAAAAMAAC9pqdJUN0xITDMxUktDT0RCUjJMVlc1OTVSRTlTRS4u!)
+
+The open-science movement is one of the best things happening in neuroscience. Hackathons, open-science room, new tools and methods, reproducibility and reliability, open access and inclusivity: the OS SIG is the place where it’s all happening!
+
+If open-science excites you and you want to help push its frontiers, please consider applying for a position in the OS SIG! 
+
+Early career researchers (Link to SIG), BIPOC, LGBTQ+ (Link to SIG), and people from outside Europe and North America especially encouraged to apply! 
+
+All OS_SIG members have reduced or no-cost registration for OHBM 2021!
+
+We have MANY opportunities to get involved with open-science and OHBM if you think nominating yourself for a position in the OS SIG is more time than you’d like to commit, please reach out to us about volunteering!
+
+We are looking for a group of interested volunteers to help coordinate the international effort of creating Hackathons and the open-science Room! Please join us!
+
+If you’re interested in taking your open-science skills to the next level, consider applying for our open-science Fellows program (more information coming in early fall).
+
+## Available positions
+
+We have many positions available (more info in link above):
+
+### Open-science Room Chairs
+
+Take the lead on coordinating the open-science room for OHBM 2021. Apply in pairs of two. Work with the OS SIG to create an inclusive space for all OHBM members! 
+
+- Number of positions: 2
+- Duration: 1 year
+
+### Hackathon Chairs
+
+Take the lead on coordinating the Hackathon for next year! Apply in pairs of two. Work with the rest of the OS SIG members to put on the best Hackathon happening anywhere.
+
+- Number of positions: 2
+- Duration: 1 year
+
+### Inclusivity Officer: 
+
+Work together with the OS SIG and OHBM community to maximize inclusivity and eliminate exclusionary barriers in open-science and OHBM.
+
+- Number of positions: ???
```

Comment:
duration of the inclusivity officer

Reply:
Inclusivity officer: 1 person, for 1 year